.pre-commit-config.yaml

```
@@ -86,7 +86,7 @@ repos:
     hooks:
       - id: bandit
         # Bandit complains about the use of assert() in tests
-        exclude: molecule/default/tests
+        exclude: molecule/(default|prerequisites_only)/tests
```

Removing this was missed in https://github.com/cisagov/ansible-role-apt-over-https/pull/16/commits/021a5225aa11b6969f9b8e6433d44b4537eb9f37:
```suggestion
        exclude: molecule/default/tests
```